pkg/booleanpolicy/violationmessages/printer/kube_event.go

```diff
@@ -82,16 +82,32 @@ func getDefaultViolationMsgHeader(event *storage.KubernetesEvent) string {
 	object := event.GetObject()
 	readableResourceName := strings.ToLower(object.Resource.String())
 
+	var singularResourceName string
+	if strings.HasSuffix(readableResourceName, "ies") {
+		singularResourceName = strings.TrimSuffix(readableResourceName, "ies")
+		singularResourceName = fmt.Sprintf("%sy", singularResourceName)
+	} else {
+		singularResourceName = strings.TrimSuffix(readableResourceName, "s")
+	}
+	singularResourceName = strings.ReplaceAll(singularResourceName, "_", " ")
+	readableResourceName = strings.ReplaceAll(readableResourceName, "_", " ")
```

Comment:
You can combine this, `auditResourceToKubeResource`,  `verbsDenyListPerResource`, and `resourceTypesAllowList` into one struct. That will make future contributions dramatically simple and the chances of missing something remain low.

Reply:
Taking a rain check on this one especially because `resourceTypesAllowList` has port forward and pod exec while the other vars don't.